ci-operator/step-registry/openshift-qe/installer/gcp/ipi/sdn/private/openshift-qe-installer-gcp-ipi-sdn-private-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: openshift-qe-installer-gcp-ipi-sdn-private
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-gcp-ipi-private-provision
+    post:
+    - chain: cucushift-installer-rehearse-gcp-ipi-private-deprovision
```

Comment:
workflow ‘cucushift-installer-rehearse-gcp-ipi-private’ is what we can refer to to create our own workflow, but please don’t directly use ‘cucushift-installer-rehearse-gcp-ipi-private-provision’ (if doing so, we don’t have to create our own workflow). There are some ref/chains we don’t need, so in our workflow we remove them, like
  - ref: enable-qe-catalogsource
  - chain: cucushift-installer-check
  - ref: cucushift-installer-check-gcp-private

Reply:
+1